src/CalcManager/Ratpack/itrans.cpp

```diff
@@ -206,7 +206,7 @@ void acosrat( PRAT *px, uint32_t radix, int32_t precision)
 {
     int32_t sgn;
 
-    sgn = (*px)->pp->sign*(*px)->pq->sign;
+    sgn = SIGN(*px);
```

Comment:
Combine the declaration at line 207 with this initialization?

Reply:
Here and elsewhere that you're using SIGN(): if it's appropriate to declare-and-assign at the same time, we should. No need to change all the surrounding variables to follow this right now, but since you're updating the `sgn` variables, let's clean them up a bit.